misc/wallaroo-up.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
 # md5 for validatiing script checksum
-MD5="2d83bfc6a87364f4abb20cc04e5d4777  -"
+MD5="d2694be4690ec1835393f30f50aa34b5  -"
 
 set -eEuo pipefail
 
 WALLAROO_UP_DEST_DEFAULT=~/wallaroo-tutorial
 WALLAROO_UP_DEST_ARG="$WALLAROO_UP_DEST_DEFAULT"
 WALLAROO_UP_INSTALL_TYPE=UNSPECIFIED
-WALLAROO_VERSION_DEFAULT=0.6.1
+WALLAROO_VERSION_DEFAULT=latest
```

Comment:
are you sure about this?

Reply:
yes, this is to avoid the test script from compiling 0.6.1 with ponyc 0.28.0. this problem wasn't present before because releases coincided with ponyc updates.